enclave-tls/src/lib/tls_wrappers/null/main.c

```diff
@@ -0,0 +1,43 @@
+#include <enclave-tls/tls_wrapper.h>
+#include <enclave-tls/err.h>
+#include <enclave-tls/log.h>
+
+extern tls_wrapper_err_t tls_wrapper_register(tls_wrapper_opts_t *);
```

Comment:
This extern should be in enclave-tls/tls_wrapper.h. Any instance code should never explicitly declare it.